R/statistic-tree.R

```diff
@@ -170,22 +170,26 @@ dpTree <- setRefClass(
 )
 
 dpTree$methods(
+    # DO NOT USE
```

Comment:
The initialize function seems like a pretty important function not to use?

Reply:
Correct. This is just marking that the entire tree statistic as currently implemented in develop, does not work. Those fixes are in a separate branch that I haven't pushed yet.